packages/mdc-tabs/tab-bar/mdc-tab-bar.scss

```diff
@@ -18,6 +18,16 @@
 @import "@material/theme/mixins";
 @import "@material/rtl/mixins";
 
+.mdc-tab-bar__indicator,
+.mdc-tab__indicator {
```

Comment:
not sure if this should be in mdc-tab.scss, or if its ok to be in here.

Reply:
It should be in mdc-tab.scss.

Since you want to share CSS properties across the two classes, you should create a private mixin and re-use it across the two files.